vendors/watteco/drivers/th_v4/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
```

Comment:
If you are obliged to have multiple js file for your decoder, and decided to use webpack. 

Here are the configurations to use: 

```
const path = require("path");

module.exports = {
  target: "node",
  mode: "production",
  entry: "./index.js",
  output: {
    filename: "main.js",
    path: path.resolve(__dirname, "."),
    library: "driver",
  },
};
```

in your case you could replace "index.js" by "th.js".